pytest_odoo.py

```diff
@@ -76,9 +79,10 @@ def pytest_cmdline_main(config):
 
         # Check the environment variables supported by the Odoo Docker image
         # ref: https://hub.docker.com/_/odoo
-        for arg in ['HOST', 'PORT', 'USER', 'PASSWORD']:
-            if os.environ.get(arg):
-                options.append('--db_%s=%s' % (arg.lower(), os.environ.get(arg)))
+        if not config.getoption("--odoo-disable-docker-env"):
```

Comment:
Why `docker` in the name ? what do you think about

```suggestion
        if not config.getoption("--odoo-disable-env"):
```

Reply:
in #57 where this was added it was mentioned that these are used in the Odoo docker container.

I'm not all that good with naming - i very much agree that what i came up with isn't good, but i think `--disable-odoo-env` as you suggested is too ambiguous and potentially confusing..

I'm very open to further suggestions though!

Reply:
what about `--odoo-ignore-automatic-db-param-from-env`?

Reply:
That's a big long.. What about `--odoo-ignore-env-db-param`? It's not as long..

Reply:
:+1: yeah and automatic was ambiguous as well